index.html

```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+  <!-- navigation section -->
+    <header>
+        <nav class="tool-bar-container">
+            <a  class="logo-icon" href="#"><img src="./Icons/logo.png" alt="">
```

Comment:
- [ ] Nice work implementing the the "Toolbar" section it looks really good :+1: However it would be nice to remove the logo from the `nav` tags. The `nav` tag suppose to contain only the navigation links.